### DISCUSSION TITLE
127.0.0.1 declined connection

### DISCUSSION THREAD

**Discussion by MichaelSchmidt1729** [symfony/symfony#49672] [Q&A]
Hi,

I've installed Symfony according to the docs. composer require webapp. I started a server. But when I open the browser to https://127.0.0.1:8000 or run symfony open:local I get "Website not accessible. 127.0.0.1 declined connection".
Im running ubuntu on WSL2.

What am I missing?

Bye

Michael

**Comment by lyrixx** [comment]
Hello,

You can check the server log, it might help you to understand what's going on
